pkg/kdump/kdump-view.jsx

```diff
@@ -41,14 +41,69 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "rhel-system-roles";
+    if (os_release.NAME.includes('Fedora')) {
+        role_name = "linux-system-roles";
+    }
+
+    let ansible = `
+---
+# ${role_name} is also available via ansible-galaxy
+- name: install ${role_name}
+  package:
+    name: ${role_name}
+    state: present
+  delegate_to: 127.0.0.1
```

Comment:
Doesn't this require `become: true`?

Reply:
Become: true when you run it as normal user?

Reply:
Yes, that's very often the case, especially for deploying to remote targets. One really shouldn't run ansible as root on a developer machine. It's already questionable to install a package locally for that, but then again this should be an inspiration for admins, not something to take without looking :grin:

Reply:
Yeah, let's do the right thing here